src/components/Tabs/Tabs.tsx

```diff
@@ -12,15 +13,45 @@ interface ITabsProps {
 }
 
 export const Tabs = ({ children }: ITabsProps) => {
+  const [showScrollButton, setShowScrollButton] = useState(false);
+  const tabContent = document.getElementById('tab-content');
+
+  const onResize = useCallback(() => {
+    tabContent && setShowScrollButton(tabContent.scrollWidth > window.innerWidth);
+  }, [tabContent]);
+
+  window.addEventListener('resize', onResize);
+
+  const scrollLeft = () => {
+    if (tabContent) {
+      tabContent.scrollLeft = tabContent.scrollLeft ? tabContent.scrollLeft - 150 : 0;
```

Comment:
If the the `scrollLeft` is zero, 'scroll left' button should be disabled.
Same for the 'scroll right' variant.
In the same way as here:
https://www.patternfly.org/v4/components/tabs/